homeassistant/components/group/__init__.py

```diff
@@ -257,12 +257,16 @@ def async_setup(hass, config):
 
     @asyncio.coroutine
     def reload_service_handler(service):
-        """Remove all groups and load new ones from config."""
+        """Remove all user-defined groups and load new ones from config."""
+        auto = list(filter(lambda e: not e.user_defined, component.entities))
```

Comment:
Please always use list comprehension when possible instead of filter + lambda. List comprehension is naturally readable/understandable.

Reply:
Thanks, noted.